src/txHashBuilder.c

```diff
@@ -290,15 +293,37 @@ void txHashBuilder_addOutput_token(
 
 	if (builder->outputData.remainingTokens == 0) {
```

Comment:
`outputData` need to be renamed to `multiassetData` or something like that, and we should carefully check they are initialized properly.